interactions/_interaction.dm

```diff
@@ -40,6 +40,7 @@ var/list/interactions
 	var/interaction_sound
 
 	var/max_distance = 1
+	var/require_ooc_consent = FALSE
```

Comment:
If you are keen to diddle this bit of code, this pile of vars could do with being flagged, but not critical.

Reply:
Not sure what you'd want changed here.